com.unity.render-pipelines.high-definition/Runtime/Debug/DebugDisplay.hlsl

```diff
@@ -130,7 +130,7 @@ float3 GetTextureDataDebug(uint paramId, float2 uv, Texture2D tex, float4 texelS
 // color is current screen color
 // color of the font to use
 // direction is 1 or -1 and indicate fixedUnormCoord block shift
-void DrawCharacter(uint asciiValue, float3 fontColor, uint2 currentUnormCoord, inout uint2 fixedUnormCoord, inout float3 color, int direction)
+void DrawCharacter(uint asciiValue, float3 fontColor, uint2 currentUnormCoord, inout uint2 fixedUnormCoord, inout float3 color, int direction, int fontTextScaleWidth)
```

Comment:
Please add this change in the upgrade guide as people may have used this function for their own purpose. (or add a function to make it an optional parameter

Reply:
ah, you have done it already :), all good